### ISSUE TITLE
Cannot unbind thumb_recenter action

### ISSUE THREAD

**Issue by thefrozncity** [LibreVR/Revive#1185]
Hello,
I have been searched the option for 1 month
really cannot find the option. where can I find the option on earth?
not in controller input binding of revive/not in revive section of openvr advanced setting.
there were no thumb_recenter action on google but what you wrote below 
"you can do this in the SteamVR input settings, just unbind the thumb_recenter action."

??? would you kindly help me? I have been searched it for 1 month because I really don't
want to bother you. but really cannot find this. 

will wait for your answer. thanks

**Comment by Armada651** [comment]
![howto](https://www.windowscentral.com/sites/wpcentral.com/files/styles/xlarge/public/field/image/2018/05/access-steamvr-binding-section-01.jpeg?itok=dU_RK57W)

**Comment by Armada651** [comment]
After that you can go the the application you want to change, press the Edit button for the Vive Controllers and then go to the "touch" tab. There you can change the Oculus Touch binding.

**Comment by thefrozncity** [comment]
first of all, I am really appreciated for you effort. but sorry really sorry but I cannot understand. some of you advice. maybe I or we missed the point.

1. I don't have oculus touch 
2, I could find edit button for vive controllers (steamvr setting -> controller binding -> edit)
3. but there wasn't touch tab. I cannot understand how can I find touch tab if I don't have oculus touch
4. and how can there be touch tab inside vive controller edit screen even if I had oculus touch

I will keep reading you comments for 2 days and now I am reading. try to find out what you mean but I really cannot understand. sorry really sorry but. kindly help me one more time. 

Thank you so much

**Comment by Armada651** [comment]
The Touch tab is there because your Vive controllers emulate an Oculus Touch controller. It's at the top between the "remote" and "xbox" tabs.

**Comment by thefrozncity** [comment]
Thank you! now I finally got it work. I couldn't find it because I didn't run the oculus home game. anyway I got robo recall work. now no need to use robo revive. but sometimes I run the game left controller rumbles without stopping. in this case, I need to quit the game and restart. 

and specific game like from other suns doesn't work with this method. I mean it still need to be swiped to move even though I unbind thumb recenter action.

Thanks!
